frontend/src/components/Modals/DeleteModal.tsx

```diff
@@ -21,8 +21,23 @@ const DeleteConfirmModal = ({open, setOpen, resourceType, resource, removeResour
                 showMessage({type: 'error', content: 'Delete failed.', key: 'delete'});
             }
         } catch (err) {
-            console.error('Error during deleting:', err);
-            showMessage({type: 'error', content: 'Delete error.', key: 'delete'});
+            if (err instanceof Error) {
```

Comment:
I hate js

Reply:
The messages have been worse. I spent an hour or two trying to come up with something better, but I couldn’t. (I’m referring to the messages in files other than the modal.)